frontend/src/container/Dashboard.jsx

```diff
@@ -518,13 +519,23 @@ class Dashboard extends React.Component {
                 />
               </div>
 
-              {props.appList.some(a => a.slug === 'agenda') && props.curWs.agendaEnabled &&
+              {props.appList.some(a => a.slug === 'agenda') && props.curWs.agendaEnabled && (
                 <AgendaInfo
-                  customClass='dashboard__agenda'
-                  introText={props.t('Use this link to access this shared space agenda from anywhere')}
+                  customClass='dashboard__section'
+                  introText={props.t("Use this link to integrate this shared space's agenda to your")}
+                  caldavText={props.t('CalDAV compatible software')}
                   agendaUrl={props.curWs.agendaUrl}
                 />
-              }
+              )}
+
+              {props.system.config.webdav_enabled && (
+                <WebdavInfo
```

Comment:
@Skylsmoi this is typically what I would expect to be generic. We should not have a AgendaInfo widget and a WebdavInfo widget but one DashboardExternalLinkInfo instantiated with specific parameters.